audiofile/core/info.py

```diff
@@ -162,13 +172,19 @@ def duration(file: str, sloppy=False) -> float:
 def samples(file: str) -> int:
     """Number of samples in audio file.
 
+    Audio files that are not WAV, FLAC, or OGG
+    are first converted to WAV,
+    before counting the samples.
```

Comment:
Can't we use `mediainfo` to get the number of samples?

Reply:
No as it reads that information only from the header and it might not be correct when reading the file.
For that reason we always convert first to WAV for counting samples for non WAV, FLAC, OGG files.

Reply:
Ok, that can be pretty slow. On the other hand, if the user needs a quick (possibly inaccurate) result she can use `round(duration * sampling_rate)`.

Reply:
Yes, and that is also the reason why we have introduced `sloppy` argument for `audiofile.duration()`.